ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -353,7 +353,7 @@ def handleAppletStateUpdateRequested(self):
         batch_processing_busy = self.batchProcessingApplet.busy
 
         self._shell.setAppletEnabled( self.dataSelectionApplet,             not batch_processing_busy )
-        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and not superpixels_available_from_file )
+        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and opDataSelection.ImageGroup[0][1].ready() and not superpixels_available_from_file )
         self._shell.setAppletEnabled( self.edgeTrainingWithMulticutApplet,  not batch_processing_busy and input_ready and superpixels_ready )
```

Comment:
While you are at it, it would be nice if you found a condition that would not enable the training applet before superpixels have been computed (as it is the case right now, `superpixels_ready`